fuzzing/oss_fuzz_build.sh

```diff
@@ -1,37 +1,30 @@
-#!/bin/bash -eu
+#!/bin/bash -eux
 
-# TODO AdamKorcz: Make the build script a gradle module
-
-./gradlew clean build -x test --refresh-dependencies
-
-CURRENT_VERSION=$(./gradlew properties --no-daemon --console=plain -q | grep "^version:" | awk '{printf $2}')
-cp sigstore-java/build/libs/sigstore-java-$CURRENT_VERSION.jar $OUT/sigstore-java.jar
-
-# Search for dependency jars from gradle local repository,
-# move them to the $OUT directory and add them to the ALL_JARS
-# variable to allow jvm to use them as class path when compiling
-# and executing the fuzzers which depends on them
-ALL_JARS=sigstore-java.jar
-for jarfile in $(find ~/.gradle/caches/modules-2/files-2.1/ \( -iname "*.jar" ! -iname "junit*" ! -iname "spotless*" ! -iname "ktlint*" ! -iname "gradle*" ! -iname "kotlin*" \))
+# build the fuzzing classes and extract dependencies into the build/fuzzRoot
+./gradlew :fuzzing:build :fuzzing:copyToFuzzOut -x test --refresh-dependencies -PfuzzOut="$OUT"
```

Comment:
I realize, the behavior was the same before this change, but maybe the script could check for the existence of `$OUT` and report clear error that it has to be set?

2. I wonder, if `-x test` is needed? It failed due to the missing `src/test/*`?

3. Why `--refresh-dependencies`? Are there any SNAPSHOTs used?

Reply:
1. $Out appears to come from the fuzzing infra, so out looks like it *must* be there
2. I think we don't care about running tests. We're just building to get the fuzzers going.
3. I didn't see that, I would assume that it shouldn't matter since all builds are fresh?

Reply:
1. $OUT is defined in the fuzzing infra while running the fuzzer in a docker image. So it must be there.
2. Yes, we don't care about running tests because in theory fuzzer are also a kind of test on its own. The exclude of test may speed up the process.
3. Yes, it may not necessary, the original reason for adding that is to make sure the fuzzer could retrieve an up to date dependency jar for execution.

Reply:
> $OUT is defined in the fuzzing infra while running the fuzzer in a docker image. So it must be there.

Ok. I was rather thinking about the people willing to run it locally (e.g. to verify the changes they made in the scripts or configuration). So probably, some extra check could be in place anyway. However, due to the missing "README", I even not sure, if it is "easy" to run it locally (even to play with for some short time) or rather the CI action is more feasible in general :-/.